avm/src/daemon/stats.rs

```
@@ -70,16 +73,29 @@ async fn get_proc_usages() -> Vec<f64> {
 async fn get_cores_total_times() -> Vec<CPUSecsV1> {
   heim_cpu::times()
     .map(|r| {
-      let cpu = r.expect("Failed to get CPU times");
-      CPUSecsV1 {
-        user: cpu.user().get::<second>(),
-        system: cpu.system().get::<second>(),
-        idle: cpu.idle().get::<second>(),
-        irq: 0.0,
-        nice: 0.0,
-        ioWait: 0.0,
-        softIrq: 0.0,
-        steal: 0.0,
+      if let Ok(cpu) = r {
+        CPUSecsV1 {
+          user: cpu.user().get::<second>(),
+          system: cpu.system().get::<second>(),
+          idle: cpu.idle().get::<second>(),
+          irq: 0.0,
+          nice: 0.0,
+          ioWait: 0.0,
+          softIrq: 0.0,
+          steal: 0.0,
+        }
+      } else {
+        error!("Failed to get CPU times");
```

we should return this error and only log it in the main daemon loop